### DISCUSSION TITLE
Label/field explorer to browse extracted labels and associated values

### DISCUSSION THREAD

**Discussion by mlcohen** [grafana/grafana#46985] [Observability]
Hi --

Submitting a request for Grafana Loki's explorer workspace to include the capability to browser extracted labels. Right now this does not appear to be possible. The Grafana Loki Explorer only allows you to browser original labels. I raise this as someone new to Loki and coming from Solarwind's Loggly. Loggly comes with a builtin panel to explore fields and values associated with those fields. (see https://documentation.solarwinds.com/en/success_center/loggly/content/getting-started/gsg-analyze.htm#Filter-on-Top-Failures). This is tremendously useful when you're not exactly sure what you are searching for or not always familiar with all of the fields/labels. Some equivalent of this in Grafana Loki explorer would be very valuable. If there is a 3rd-party plugin that does provide this capability in the explorer workspace, I'd happy to use that. I have not been able to find such a plugin.

I originally posted about this in the Grafana community dashboard and was advised to post here. (see https://community.grafana.com/t/log-entry-field-value-tabulation-in-loki-explorer-workspace/62289)

**Comment by ifrost** [comment]
cc @grafana/observability-logs-and-traces

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
